### ISSUE TITLE
systemd is causing mount locks on CentOS AMI

### ISSUE THREAD

**Issue by argenet** [scylladb/scylla-ami#37]
The issue has been seen on a Scylla CentOS AMI-deployed VM where `df -h` hangs at
```
stat("/", {st_mode=S_IFDIR|0555, st_size=4096, ...}) = 0
stat("/proc/sys/fs/binfmt_misc",
```

This is due to the following issue fixed in `systemd-234`:
https://github.com/systemd/systemd/pull/5916

The issue is known to be triggered by Prometheus `node-explorer`: https://github.com/coreos/coreos-kubernetes/issues/739#issuecomment-256248527

More info on the issue can be found in this thread: https://github.com/coreos/bugs/issues/1630

**Comment by argenet** [comment]
One way is to upgrade the `systemd` package used by CentOS AMI.
Currently, it is `systemd-219` which is extremely outdated (dating back to 2015).
We may consider using Facebook RPMs that backported `systemd` to CentOS 7:
https://github.com/facebookincubator/rpm-backports
See more on that: https://maciej.lasyk.info/2016/Dec/16/systemd-231-latest-in-centos-7-thx-to-facebook/

Another way is to apply some workaround that can be deduced from the threads above.

**Comment by syuu1228** [comment]
RHEL/CentOS systemd looks very old version, but it backports many recent patches.
So I checked the patch for systemd/systemd#5916 is merged on RHEL systemd or not, then I found it seems backported on RHEL/CentOS systemd from v219-54 (https://github.com/lnykryn/systemd-rhel/commit/d9df4d0ed03f56036b04e19a8e6be2c028c4a72e).
On recent release of Scylla AMI we use CentOS 7.5, witch contains systemd-219-57.
So I think the bug should fixed now, and looks like does not reproduce on it.
